ARK Server Manager/Lib/ServerRCON.cs

```diff
@@ -370,13 +370,45 @@ private void HandleCommand(ConsoleCommand command)
                     }
                 }).DoNotWait();
             }
+            else
+            {
+                var arkData = await ArkData.ArkDataContainer.CreateRemoteAsync();
+                arkData.SteamLoaded = true;
+                //await arkData.LoadSteamAsync(Config.Default.SteamAPIKey);
+                TaskUtils.RunOnUIThreadAsync(() =>
+                {
+                    foreach (var playerData in arkData.Players)
+                    {
+                        var playerToUpdate = this.Players.FirstOrDefault(p => p.SteamId == Int64.Parse(playerData.SteamId));
+                        if (playerToUpdate != null)
+                        {
+
+                            playerToUpdate.AvatarImage = new BitmapImage(new Uri(playerData.AvatarUrl));
```

Comment:
The only reason I did it this way is because for some reason your Async function that was doing the same thing in the background would never callback. I put a breakpoint there and let it sit for 5 minutes and nothing came of it.